python/ray/tune/tests/test_tuner.py

```diff
@@ -322,23 +322,26 @@ def test_tuner_fn_trainable_checkpoint_at_end_none():
 
 
 @pytest.mark.parametrize("runtime_env", [{}, {"working_dir": "."}])
-def test_tuner_no_chdir_to_trial_dir(runtime_env):
+def test_tuner_no_chdir_to_trial_dir(runtime_env, tmpdir):
     """Tests that setting `chdir_to_trial_dir=False` in `TuneConfig` allows for
     reading relatives paths to the original working directory.
     Also tests that `session.get_trial_dir()` env variable can be used as the directory
     to write data to within the Trainable.
     """
-    if ray.is_initialized():
-        ray.shutdown()
-    ray.init(num_cpus=1, runtime_env=runtime_env)
+    old_cwd = os.getcwd()
+    os.chdir(tmpdir)
 
     # Write a data file that we want to read in our training loop
     with open("./read.txt", "w") as f:
         f.write("data")
 
+    if ray.is_initialized():
+        ray.shutdown()
+    ray.init(num_cpus=1, runtime_env=runtime_env)
+
```

Comment:
Hm, maybe we can copy the `shutdown_only` fixture from other (core) tests and add it to the other tests? Instead of handling the test environment here explicitly